internal/humanitec_go/client/deployments.go

```diff
@@ -53,6 +58,13 @@ func (api *apiClient) StartDeployment(ctx context.Context, orgID, appID, envID s
 			return &res, nil
 		}
 
+	case http.StatusConflict:
+		if retry {
```

Comment:
I'd suggest to make `--retry N` configurable number of attempts.

Also, following this logic, make a delay configurable too, e.g. `--retry-delay S`, where S are seconds. Default is 5 seconds.

Reply:
Who knows about this 15 min cap somewhere in Humanitec system? Does it mean we won't be able to change this internal delay now, as it would break someone's deployment pipeline?

We should make all settings configurable explicitly.

Reply:
> Does it mean we won't be able to change this internal delay now, as it would break someone's deployment pipeline?

Yes, changing this could potentially break deployments, but that is already the case without this PR.

> Who knows about this 15 min cap somewhere in Humanitec system?

I don't know, but it should definitely be documented as a limit.